### PR TITLE
DOCS-2250 Lint Account Management section

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
Lint Account Management section

### Motivation
New Vale Linter rules

### Preview
https://docs-staging.datadoghq.com/ruth/docs-2250/account_management

### Additional Notes
N/A

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/rum.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Real User Monitoring billing FAQ
-kind: faq
+title: Real User Monitoring Billing
+kind: documentation
```
Comment (kayayarai) [inline content/en/account_management/billing/rum.md:3]: Are we sure we want to change this? i have a feeling it was set to `faq` for a reason. @apigirl ?

File: content/en/account_management/rbac/permissions.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ further_reading:
       text: 'Manage your permissions with the Permission API'
 ---
 
-Once your roles are created, assign or remove permission to this role directly by [updating the role in the Datadog application][1], or through the [Datadog Permission API][2]. Find below a list of available permissions.
+After creating roles assign or remove permission to this role directly by [updating the role in Datadog][1], or through the [Datadog Permission API][2]. Find below a list of available permissions.
```
Comment (kayayarai) [inline content/en/account_management/rbac/permissions.md:16]:
```suggestion
After creating roles, assign or remove permission to this role directly by [updating the role in Datadog][1], or through the [Datadog Permission API][2]. Find below a list of available permissions.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Just a couple edits and questions

File: content/en/account_management/billing/rum.md (hunk 1 above)
Comment (ruthnaebeck) [reply content/en/account_management/billing/rum.md:3]: Possibly, but the file should be in the FAQ folder if it's an FAQ.

Comment (apigirl) [reply content/en/account_management/billing/rum.md:3]: this change is fine

**Review by kayayarai — APPROVED** [review]

**Pull request merged**